pkg/scheduler/preemption/preemption.go

```diff
@@ -96,8 +96,24 @@ func candidatesFromCQOrUnderThreshold(candidates []*workload.Info, clusterQueue
 	return result
 }
 
+type Target struct {
+	Wl     *workload.Info
```

Comment:
```suggestion
	WorkloadInfo     *workload.Info
```
or

```suggestion
	Workload     *workload.Info
```

If we want to expose this field to the external packages, a predictable field name would be better.